packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java

```diff
@@ -341,15 +322,15 @@ void sendInitialized() {
       event.put("event", "initialized");
       event.put("duration", exoPlayer.getDuration());
 
-      if (exoPlayer.getVideoFormat() != null) {
-        Format videoFormat = exoPlayer.getVideoFormat();
+      Format videoFormat = unstableGetVideoFormat(exoPlayer);
```

Comment:
Can we use `getVideoSize` here instead of `getVideoFormat`? It appears that we just need width, height, and rotation, all of which [`VideoSize`](https://developer.android.com/reference/androidx/media3/common/VideoSize) has.

Reply:
thanks, updated

not sure I've applied rotation changes correctly though